src/utils/color-set.ts

```diff
@@ -0,0 +1,8 @@
+import { ColorSetGroup } from './color-set-group.enum';
+
+export interface ColorSet {
+  name: string;
+  selectable: boolean;
+  group: ColorSetGroup;
```

Comment:
Could probably be changed for `group: 'Continuous' | 'Ordinal',` if the group is not being used elsewhere, but it's always better to be safer.